packages/studio-base/src/components/AppSettingsDialog/AppSettingsDialog.tsx

```diff
@@ -144,7 +144,10 @@ const aboutItems = new Map<
     {
       subheader: "Legal",
       links: [
-        { title: "License terms", url: "https://foxglove.dev/legal/studio-license" },
+        {
+          title: "License terms",
+          url: "https://github.com/bmw-software-engineering/lichtblick/blob/main/LICENSE",
+        },
         { title: "Privacy policy", url: "https://foxglove.dev/legal/privacy" },
```

Comment:
The privacy will continue pointing to foxglove?

Reply:
Done, thanks for the hint